### PR TITLE
Log JS SDK library

### PR DESCRIPTION
**Pull request by nbierdeman**
### Description

Herein lies part of the effort charged by DTPPCPSDK-1482.

### Why are we making these changes?

This sends `jsSdkLibrary` to FPTI & SFX for 1st and 2nd render.

### CODE REVIEW

**Comment by wsbrunson** [comment]
we are going to add the default `direct-script` integration source type here?

**Review by ravishekhar — APPROVED** [review]

**Comment by nbierdeman** [comment]
> we are going to add the default `direct-script` integration source type here?

@wsbrunson, I'm going to wait on merging this PR, because I'm not totally clear yet on where we're going to enforce values. I'm thinking we should enforce the JS library values in the JS libraries themselves and `RAW_SCRIPT` here. Will consult with you as I make more progress in react-paypal-js and paypal-js.

**Comment by nbierdeman** [comment]
codecov is wanting me to write tests for alphabetizing the FPTI keys. 👇 🙃

File: src/script.test.js [hunk 1]
```diff
@@ -253,6 +255,16 @@ describe(`script cases`, () => {
...
 
+  it("should successfully get js sdk library", () => {
+    const jsSdkLibrary = JS_SDK_LIBRARIES.REACT_PAYPAL_JS;
+    const mockElement = makeMockScriptElement(mockScriptSrc);
+    mockElement.setAttribute(SDK_SETTINGS.JS_SDK_LIBRARY, jsSdkLibrary);
+    // $FlowIgnore
+    getCurrentScript.mockReturnValue(mockElement);
+
+    expect(getJsSdkLibrary()).toEqual(jsSdkLibrary);
+  });
```
Comment (jshawl) [inline src/script.test.js:266]: nice!

File: src/tracking.js [hunk 2]
```diff
@@ -107,22 +109,23 @@ export function setupLogger() {
...
-      [FPTI_KEY.FEED]: FPTI_FEED.PAYMENTS_SDK,
-      [FPTI_KEY.DATA_SOURCE]: FPTI_DATA_SOURCE.PAYMENTS_SDK,
       [FPTI_KEY.CLIENT_ID]: getClientID(),
-      [FPTI_KEY.SELLER_ID]: mID && mID.toString(),
-      [FPTI_KEY.SESSION_UID]: getSessionID(),
-      [FPTI_KEY.REFERER]: window.location.host,
-      [FPTI_KEY.LOCALE]: `${lang}_${country}`,
+      [FPTI_KEY.CONTEXT_CORRID]: getCorrelationID(),
+      [FPTI_KEY.DATA_SOURCE]: FPTI_DATA_SOURCE.PAYMENTS_SDK,
+      [FPTI_KEY.FEED]: FPTI_FEED.PAYMENTS_SDK,
```
Comment (jshawl) [inline src/tracking.js:115]: 🔥 (🔤)

**Review by jshawl — APPROVED** [review]
:shipit:

**Pull request merged**